iot/api-client/end_to_end_example/cloudiot_pubsub_example_server.py

```diff
@@ -56,10 +56,9 @@
 
 class Server(object):
     """Represents the state of the server."""
-
-    def __init__(self, service_account_json):
-        credentials = ServiceAccountCredentials.from_json_keyfile_name(
-            service_account_json, API_SCOPES)
+def __init__(self, service_account_json):        
```

Comment:
I'm not sure this is the intended indentation as the __init__ block should be inside the Server class.